sentry_sdk/integrations/huey.py

```diff
@@ -144,34 +139,31 @@ def patch_execute():
     # type: () -> None
     old_execute = Huey._execute
 
+    @ensure_integration_enabled(HueyIntegration, old_execute)
     def _sentry_execute(self, task, timestamp=None):
         # type: (Huey, Task, Optional[datetime]) -> Any
-        hub = Hub.current
+        scope = Scope.get_isolation_scope()
 
-        if hub.get_integration(HueyIntegration) is None:
-            return old_execute(self, task, timestamp)
-
-        with hub.push_scope() as scope:
```

Comment:
We should probably have an isolation scope fork in the new API, to match the `push_scope` behavior